definitions/tools/manta_somatic.cwl

```diff
@@ -48,6 +52,11 @@ inputs:
         inputBinding:
             position: -7
             prefix: "--outputContig"
+    jobs:
```

Comment:
Do we want to define the number of jobs using an input parameter, or should we resolve the command line argument from the `ResourceRequirement`? Here is an example:
https://github.com/genome/analysis-workflows/blob/9ad798f18104eb4a9c6c118f42691b9f5d9efd19/definitions/tools/stringtie.cwl#L16